airbyte-integrations/connectors/source-airtable/acceptance-test-config.yml

```diff
@@ -19,14 +19,21 @@ acceptance_tests:
   discovery:
     tests:
       - config_path: "secrets/config.json"
+        # the static `string` schema is replaced with casted to JSONSchema
+        backward_compatibility_tests_config:
+          disable_for_version: "1.0.2"
```

Comment:
This is more of a clarity question for @alafanechere - `Disable the backward compatibility test for a specific version (expects a version following semantic versioning).` I'd expect that to mean that for version V we don't check to see if its backward compatible with version V-1 where V-1 means the version released before V. In that case it'd be setting it to `2.0.0` here. We actually set it for the previous version?

Reply:
Yes, the `1.0.2` is the latest for now, it's published to our Dockerhub and contains OAuth implementation,  but it's not currently released to Cloud, we've pinned the version to be `1.0.1` to allow the server components to update before we release the connector with OAuth to Cloud. However, the `2.0.0` will include all the previous changes + this big `casting` functionality, which could be treated as a `Breaking Change` and `Major` release. All subsequent versioning will be smooth on 2.0.x and semantic versioning is considered. But thanks for the question!

Reply:
Cool, yup just wanted to check - code references the previous connector version 
`if previous_connector_version == inputs.backward_compatibility_tests_config.disable_for_version:` so this does make sense - just not how I interpreted the parameter.